src/Auxiliary/Aux_Check_Parameter.cpp

```diff
@@ -1212,14 +1285,23 @@ void Aux_Check_Parameter()
    if ( OPT__INIT == INIT_BY_FILE )
       Aux_Message( stderr, "WARNING : currently we don't check MIN_DENS for the initial data loaded from UM_IC !!\n" );
 
+
+#  if ( WAVE_SCHEME == WAVE_FD )
+
 #  elif ( WAVE_SCHEME == WAVE_GRAMFE ) // #  if ( WAVE_SCHEME == WAVE_FD )
 
+   if ( OPT__FIXUP_FLUX )
+      Aux_Message( stderr, "WARNING : OPT__FIXUP_FLUX will not be applied when %s is on !!\n",
+                   "WAVE_SCHEME == WAVE_GRAMFE" );
```

Comment:
Revise the warning message since, IIUC, `OPT__FIXUP_FLUX` will still be applied if the finer level still uses the fluid scheme.